Source/AudioSettingsDialog.cpp

```
@@ -574,6 +618,49 @@ void AudioSettingsDialog::onResize(int width, int height) {
 }
 
 void AudioSettingsDialog::onUpdate(double deltaTime) {
+    // Check for pending errors from audio thread
+    {
+        std::lock_guard<std::mutex> lock(m_errorMutex);
+        if (m_hasPendingError) {
+            m_errorMessage = m_pendingErrorMessage;
+            m_errorMessageAlpha = 1.0f; // Fade in
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)